### DISCUSSION TITLE
My GitHub Page no longer builds due to newly introduced GitHub Action "bot"

### DISCUSSION THREAD

**Discussion by philipmjohnson** [community/community#23598] [Other Feature Feedback, Questions, & Ideas]
<p>I have been using Docusaurus to create a static site that is hosted via GitHub Pages successfully for over a year.  The repository is here:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/agilegardenclub/agilegardenclub.github.io">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/agilegardenclub/agilegardenclub.github.io" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/290288/181078140-bcf2faec-af05-4c2f-9bce-8ec37d5b6427.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d36b36d8c026b803c5cbe4a369772be06b172b71_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d36b36d8c026b803c5cbe4a369772be06b172b71_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/3/d36b36d8c026b803c5cbe4a369772be06b172b71.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d36b36d8c026b803c5cbe4a369772be06b172b71_2_10x10.png"></div>

<h3><a href="https://github.com/agilegardenclub/agilegardenclub.github.io" target="_blank" rel="noopener nofollow ugc">GitHub - agilegardenclub/agilegardenclub.github.io: Documentation site for...</a></h3>

  <p>Documentation site for Agile Garden Club. Contribute to agilegardenclub/agilegardenclub.github.io development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>There is a “source” branch, containing the Docusaurus sources. When I run <code>npm run deploy</code> locally, the static site is built and pushed to the “main” branch.</p>
<p>This all worked great until this morning, when I tried to build and deploy a minor change to the site.  I discovered that there is now a GitHub Action that is triggered (apparently by default, because I don’t have any workflows specified for this repository) to build and deploy the site. The build succeeds (because there’s a .nojekyll), but the deploy step fails:</p>
<aside class="onebox githubactions" data-onebox-src="https://github.com/agilegardenclub/agilegardenclub.github.io/actions/runs/1659919345">
  <header class="source">

      <a href="https://github.com/agilegardenclub/agilegardenclub.github.io/actions/runs/1659919345" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Commit">

      <svg width="24" height="24" class="github-icon github-icon-failure" viewbox="0 0 24 24" aria-hidden="true"><path d="M1 12C1 5.925 5.925 1 12 1s11 4.925 11 11-4.925 11-11 11S1 18.075 1 12zm8.036-4.024a.75.75 0 00-1.06 1.06L10.939 12l-2.963 2.963a.75.75 0 101.06 1.06L12 13.06l2.963 2.964a.75.75 0 001.061-1.06L13.061 12l2.963-2.964a.75.75 0 10-1.06-1.06L12 10.939 9.036 7.976z"></path></svg>

  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/agilegardenclub/agilegardenclub.github.io/actions/runs/1659919345" target="_blank" rel="noopener nofollow ugc">Deploy website - based on acc27db0bb2c8655554c37014eca537c2b7342e9</a>
    </h4>

      <div class="github-info">
        pages build and deployment <span class="github-run-number">#2</span>
      </div>
  </div>
</div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>The error appears to be:</p>
<pre><code class="lang-auto">Failed to create deployment for 4447f087281841f19cefce4e805308029ffbd603.
{"message":"Invalid actions OIDC token due to No keys from key endpoint match the id token, validate around 1641409671.","documentation_url":"https://docs.github.com/rest/reference/repos#create-a-github-pages-deployment"}
</code></pre>
<p>How do I fix this? Some potential solutions:</p>
<ol>
<li>Is there any way to not run these GitHub Actions?</li>
<li>I don’t understand the error message. Is there a way to fix it so that deployment succeeds?</li>
</ol>
<p>Thank you for any guidance.</p>

**Comment by yoannchaudet** [comment]
<p>Hello,</p>
<p>We had an unfortunate incident this morning that temporarily prevented deployments from completing.</p>
<p>Deployments started to go though again at ~ 11.21am PST and we status back a little bit later once we were fully recovered. This was caused to some internal secrets rotation.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://www.githubstatus.com/incidents/vjq8l5y6kqbj">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/14911070/181078142-74af3611-8f49-479e-9845-1052464e4ebe.png" class="site-icon" width="32" height="32">

      <a href="https://www.githubstatus.com/incidents/vjq8l5y6kqbj" target="_blank" rel="noopener">githubstatus.com</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/14911070/181078143-9c3ebd45-db25-4a98-8820-150335df4d66.png" class="thumbnail onebox-avatar" width="120" height="120">

<h3><a href="https://www.githubstatus.com/incidents/vjq8l5y6kqbj" target="_blank" rel="noopener">Incident with GitHub Pages</a></h3>

  <p>GitHub's Status Page - Incident with GitHub Pages.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
